core/templates/dev/head/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -78,7 +137,12 @@ angular.module('oppia').directive('contributionsAndReview', [
                   ' / ' + details.chapter_title),
                 labelText: SUGGESTION_LABELS[suggestion.status].text,
                 labelColor: SUGGESTION_LABELS[suggestion.status].color,
-                actionButtonTitle: (ctrl.reviewTabActive ? 'Review' : 'View')
+                actionButtonTitle: (
+                  ctrl.activeContributionTab ===
+                  ctrl.SUGGESTION_TYPE_TRANSLATE ?
```

Comment:
Doesn't this change the existing behavior? We would like to change to review tab only if "reviewTabActive" is true right?